### PR TITLE
Code improvements

### PR DESCRIPTION
**Pull request by Neustradamus**
copyright, include, class, namespace parts.

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -500,8 +499,7 @@ cat >"$1/modules.cpp" <<EOT
 name: Qt >= 4.2
 -----END QCMOD-----
 */
-class qc_qt42 : public ConfObj
-{
+class qc_qt42 : public ConfObj {
```
Comment (Ri0n) [inline configure]: I don't know what's you intention in changing auto-generating files but this change  anyway doesn't respect our code style. original code was valid

File: qcm/buildmode.qcm [hunk 2]
```diff
@@ -18,8 +18,7 @@ bool qc_buildmode_debug = false;
 bool qc_buildmode_framework = false;
 bool qc_buildmode_separate_debug_info = false;
 
-class qc_buildmode : public ConfObj
-{
+class qc_buildmode : public ConfObj {
```
Comment (Ri0n) [inline qcm/buildmode.qcm]: see my previous comment

File: src/irisnet/appledns/appledns.cpp [hunk 3]
```diff
@@ -152,9 +150,7 @@ static XMPP::NameRecord importQDnsSdRecord(const QDnsSd::Record &in)
 }
 
 namespace {
-
-class QDnsSdDelegate
-{
+class QDnsSdDelegate {
```
Comment (Ri0n) [inline src/irisnet/appledns/appledns.cpp]: basically you did the same wrong change everywhere

File: configure (hunk 1 above)
Comment (Neustradamus) [reply configure]: -> https://google.github.io/styleguide/cppguide.html :)

Comment (Vitozz) [reply configure]: What does this document have to do with the psi project?  Project has it's own approved codestyle.

Comment (Neustradamus) [reply configure]: -> https://isocpp.org/wiki/faq/coding-standards

Comment (Vitozz) [reply configure]: https://wiki.qt.io/Qt_Coding_Style

**Pull request merged**